crates/sui-indexer-alt/src/pipeline/concurrent/mod.rs

```diff
@@ -136,7 +133,8 @@ impl<H: Handler> Batched<H> {
     /// The batch is full if it has more than enough values to write to the database, or more than
     /// enough watermarks to update.
     fn is_full(&self) -> bool {
-        self.values.len() >= H::MAX_CHUNK_ROWS || self.watermark.len() >= MAX_WATERMARK_UPDATES
+        let max_chunk_rows = i16::MAX as usize / H::Value::FIELD_COUNT;
```

Comment:
nit: Seeing as this is used in two places, maybe you can introduce a function for it in this module? That way we don't have to duplicate it, but we also don't allow people to modify it.

```rust
const fn max_chunk_rows<H: Handler>() -> usize {
  i16::MAX as usize / H::Value:FIELD_COUNT
}
```